etc/sync-function-loader.js

```diff
@@ -44,15 +44,15 @@ exports.load = function(docDefinitionFilename, baseDir) {
   syncDocDefn = syncDocDefn.replace(/importDocumentDefinitionFragment\s*\(\s*"((?:\\"|[^"])+)"\s*\)/g, readDocDefinitionFragment)
     .replace(/importDocumentDefinitionFragment\s*\(\s*'((?:\\'|[^'])+)'\s*\)/g, readDocDefinitionFragment);
 
-  // Load the document definitions into the sync function template and then escape any occurrence of the backtick character so the sync
-  // function can be used in a Sync Gateway configuration file multiline string
-  var syncFunc = syncFuncTemplate.replace('%SYNC_DOCUMENT_DEFINITIONS%', function() { return syncDocDefn; })
-    .replace(/`/g, function() { return '\\`'; });
```

Comment:
Unrelated to the primary purpose of this pull request, I moved this statement further below to be with the rest of the fluent API calls that are chained together after running the sync function through indent.js.